src/test/scala/org/kibanaLoadTest/helpers/Helper.scala

```diff
@@ -293,4 +295,41 @@ object Helper {
     }
     file
   }
+
+  def checkFilesExist(paths: Path*): Unit = {
+    paths.foreach(path => {
+      val file = new File(path.toString)
+      if (!file.exists()) {
+        throw new RuntimeException(s"File does not exist: ${file.getPath}")
+      }
+    })
+  }
+
+  def readArchiveFile(filePath: Path): List[String] = {
+    checkFilesExist(filePath)
+    val inputStream = if (filePath.toString.endsWith(".gz")) {
+      new InputStreamReader(
+        new GZIPInputStream(new FileInputStream(filePath.toFile)),
+        "UTF-8"
+      )
+    } else if (filePath.toString.endsWith(".json")) {
```

Comment:
Given that this method can handle both archives and JSON files should it be renamed to convey that?